eval/pal_runner.py

```
@@ -0,0 +1,123 @@
+        return []
+
+    text = body["choices"][0]["message"]["content"] or ""
+    text = re.sub(r"^```(?:json)?\s*\n?", "", text.strip())
+    text = re.sub(r"\n?```\s*$", "", text.strip())
+    try:
+        parsed = json.loads(text)
+        if isinstance(parsed, list):
+            return parsed
+        if isinstance(parsed, dict) and "findings" in parsed:
+            return parsed["findings"]
+        return []
```

Ruff (0.15.12)

[warning] Consider moving this statement to an `else` block

(TRY300)